### ISSUE TITLE
Access denied

### ISSUE THREAD

**Issue by gofortiss** [shobhit99/InstagramAPI#1]
When i would like to access to profile picture or post, instagram say "Access denied"

**Comment by shobhit99** [comment]
I think you are trying to access data of private profile. this code only works for public profiles

**Comment by shobhit99** [comment]
are you talking about csrf token error ? 
on localhost and some webservers ( 000webhost for e.g ) requests are working as expected. but on namecheap and other servers it's not working. i'll look into it.

**Comment by gofortiss** [comment]
Have you recently tried on localhost ? Because I tried on localhost and it did not work

**Comment by shobhit99** [comment]
yes it's working for me. try with different ip. probably instagram blacklisted ip or something.
